### ISSUE TITLE
nuxt.js npm install error

### ISSUE THREAD

**Issue by ytetsuro** [inocop/docker-firebase#2]
The following error occurred when executing the following command.

command
```
# cd chat-sample
# npm install
```

error
```
npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! path /var/app/sample-chat/node_modules/opencollective/dist/bin/opencollective.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/var/app/sample-chat/node_modules/opencollective/dist/bin/opencollective.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-05-05T09_11_48_501Z-debug.log
```

I think that it is an error depending on the platform that you are using with docker.

My environment is below.

OS: macOS 10.12.6
Docker Version: Docker version 17.12.0-ce, build c97c6d6

**Comment by ytetsuro** [comment]
https://stackoverflow.com/questions/40226745/npm-warn-notsup-skipping-optional-dependency-unsupported-platform-for-fsevents?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

Can you use `fsevents` on other than macOS?
Is this command a command to be executed by HostOS?

**Comment by ytetsuro** [comment]
I ran successfully at HostOS and succeeded.

```shell
$ npm install

> fsevents@1.2.3 install /Volumes/Transcend/dev/pr/docker-firebase/src/sample-chat/node_modules/fsevents
> node install

[fsevents] Success: "/Volumes/Transcend/dev/pr/docker-firebase/src/sample-chat/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile

> nuxt@1.4.0 postinstall /Volumes/Transcend/dev/pr/docker-firebase/src/sample-chat/node_modules/nuxt
> opencollective postinstall || exit 0


                          Thanks for installing nuxt 🙏
                 Please consider donating to our open collective
                        to help us maintain this package.

                           Number of contributors: 113
                              Number of backers: 84
                            Annual budget: US$ 17,417
                            Current balance: US$ 3,345

           👉  Donate: https://opencollective.com/nuxtjs/donate

npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.

added 75 packages from 31 contributors in 23.249s
```

**Comment by ytetsuro** [comment]
fixed #1
